formatTest/typeCheckedTests/expected_output/reasonComments.re

```diff
@@ -61,36 +73,31 @@ let testingEndOfLineComments = [
   "Item 3", /* Comment For Third Item */
   "Item 4" /* Comment For Fourth Item - but before trailing comma */
   /* Comment after last item in list. */
-]
-/* Comment after rbracket */;
+] /* Comment after rbracket */;
 
 /* The space between ; and comment shoudn't matter */
 let testPlacementOfTrailingComment = [
   "Item 0" /* */
   /* Comment after last item in list. */
-];
+]; /* Comment after semi */
```

Comment:
All comments after semicolon are perfectly attached to the entire let statement (Step 2) and it naturally becomes end of line comment. 

I like the idea of formatting it to the top of statement and I think it is making more sense the more I think about it:
```
  /* Comment after semi */
 let testPlacementOfTrailingComment = [
   "Item 0" /* */
   /* Comment after last item in list. */
];
```